### PR TITLE
docs(homepage): link to typescript version

### PR DESCRIPTION
**Pull request by michaelbrewer**
**Issue #, if available:** https://awslabs.github.io/aws-lambda-powertools-typescript/latest/

## Description of changes:

Add link to typescript version

**Checklist**

* [X] [Meet tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
* [X] Update tests
* [X] Update docs
* [X] PR title follows [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-python/blob/376ec0a2ac0d2a40e0af5717bef42ff84ca0d1b9/.github/semantic.yml#L2)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

-----
[View rendered README.md](https://github.com/gyft/aws-lambda-powertools-python/blob/doc-typescript-link/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,10 +43,9 @@ With [pip](https://pip.pypa.io/en/latest/index.html) installed, run: ``pip insta
 * Structured logging initial implementation from [aws-lambda-logging](https://gitlab.com/hadrien/aws_lambda_logging)
 * Powertools idea [DAZN Powertools](https://github.com/getndazn/dazn-lambda-powertools/)
 
-
 ## Connect
 
-* **AWS Developers Slack**: `#lambda-powertools`** - **[Invite, if you don't have an account](https://join.slack.com/t/awsdevelopers/shared_invite/zt-yryddays-C9fkWrmguDv0h2EEDzCqvw)**
+* **AWS Developers Slack**: `#lambda-powertools` - **[Invite, if you don't have an account](https://join.slack.com/t/awsdevelopers/shared_invite/zt-yryddays-C9fkWrmguDv0h2EEDzCqvw)**
```
Comment (heitorlessa) [inline README.md:48]: Linting? Please revert

**Review by heitorlessa — CHANGES_REQUESTED** [review]
Revert linting

File: README.md (hunk 1 above)
Comment (michaelbrewer) [reply README.md:48]: Ah. I was confused by how this looked. With the "**" after the channel name.

Comment (michaelbrewer) [reply README.md:48]: What is looks like now:

Screen Shot 2022-01-11 at 11 05 57 PM

Comment (heitorlessa) [reply README.md:48]: That's odd, I had fixed that before that's I got confused. Maybe it accidentally made it back in one of the previous PRs.

**Review by heitorlessa — APPROVED** [review]

**Pull request merged**